### PR TITLE
Closes #61 "Route summary reporting error"

### LINKED ISSUE #61: Route summary reporting error
In the "Distance" box of a driving route a time of 22 min 43 sec is calculated.
The "Route Summary" on the map when the mouse is over the route shows this as 1h 22 min.

Difficult to describe in words so see the attached screen grab.

screenshot_20230218_215159

### PR DESCRIPTION
**Pull request by rakeshhotker**
## 🛠️ Fixes Issue
"Route summary reporting error"

Closes #61

## 👨‍💻 Changes proposed
 
1) Created a durationFormatter.js  file in utils folder
2) In the durationFormatter.js file created the formatDuration function which can be imported for use anywhere required
3) Importing the formatDuration function in Summary.jsx and Map.jsx to make sure the time displayed is same. Hence fixing the issue

## 📄 Note to reviewers
Please let me know of any modifications required.

## 📷 Screenshots
image

### CODE REVIEW

**Comment by rakeshhotker** [comment]
Sure, I'll do the necessary changes, thanks for your inputs.

File: src/utils/durationFormatter.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// const formatDuration = (durationInSeconds) => {
+//   const date = new Date(durationInSeconds * 1000)
+//   const days = date.getDate() - 1 > 0 ? date.getDate() - 1 + 'd ' : ''
+//   const hours = date.getHours() > 0 ? date.getHours() + 'h ' : ''
+//   const minutes = date.getMinutes() > 0 ? date.getMinutes() + 'min' : ''
+//   return days + hours + minutes
+// }
```
Comment (nilsnolde) [inline src/utils/durationFormatter.js:7]: no comments pls

**Review by nilsnolde — CHANGES_REQUESTED** [review]
thanks. can you change the filename to `date_time.js`? I'd like to avoid file-sprawl where each tiny function gets its own file.

**Comment by nilsnolde** [comment]
Jep ok thanks

**Pull request merged**